torch/_utils.py

```diff
@@ -705,6 +705,7 @@ def reraise(self):
         raise exception
 
 
+@functools.lru_cache(None)
```

Comment:
This is not ok. The custom backend is expected to be loaded after the fact. We cannot cache the result for it.

Reply:
Here I have an assumption: typically, after importing torch, immediately follow with the import statement for the custom backend and each custom backend is mutually exclusive. Seems like this assumption is too strong and not flexible. I will remove this cache.

Reply:
In my head, the dangerous case is any part of PyTorch or user lib that would call this library on import and thus trigger the caching before the custom backend is loaded.

Reply:
make sense.